qiskit_experiments/library/characterization/analysis/zz_ramsey_analysis.py

```diff
@@ -0,0 +1,253 @@
+        :class:`ZZRamsey` experiment. The two functions share the frequency,
+        amplitude, decay constant, baseline, and phase parameters.
+
+        .. math::
+
+            y_0 = {\rm amp} \cdot e^{-x/\tau} \cos\left(2 \pi\cdot {\rm freq - zz / 2}\cdot x + {\rm phase}\right) + {\rm base} \\
+
+            y_1 = {\rm amp} \cdot e^{-x/\tau} \cos\left(2 \pi\cdot {\rm freq + zz / 2}\cdot x + {\rm phase}\right) + {\rm base}
+
+        If the drive frequency of the measured qubit (qubit 0) is calibrated to
+        be halfway between the frequency of its frequency when the other qubit
+        (qubit 1) is in the ground state and its frequency when the other qubit
+        is in the excited state, :math:`freq` is the same as the "fake"
+        frequency :math:`f` mentioned in :class:`ZZRamsey`. Often,
+        the drive frequency is calibrated to the frequency of the measured
+        qubit with the other qubit in the ground state. In this case,
+        :code:`f = freq - zz / 2`.
+
+    # section: fit_parameters
+
+        defpar \rm amp:
+            desc: Amplitude of the sinusoidal curves.
+            init_guess: Half of the maximum y value less the minimum y value.
+            bounds: [0, the peak to peak range of the data]
+        defpar \tau:
+            desc: The exponential decay of the curve amplitudes.
+            init_guess: Inferred by comparing the peak to peak amplitude for
+                longer delay values with that of shorter delay values and
+                assuming an exponential decay in amplitude.
+            bounds: [1/4 of the typical time spacing,
+                10 times the maximum delay time].
+        defpar \rm base:
+            desc: Base line of both series.
+            init_guess: The average of the data, excluding outliers
+            bounds: [the minimum amplitude less the peak to peak of the data,
+                the maximum amplitude plus the peak to peak of the data]
+        defpar \rm freq:
+            desc: Average frequency of both series.
+            init_guess: The average of the frequencies with the highest power
+                spectral density for each series.
+            bounds: [0, the Nyquist frequency of the data].
+        defpar \rm zz:
+            desc: The :math:`ZZ` value for the qubit pair. In terms of the fit,
+                this is frequency difference between series 1 and series 0.
+            init_guess: The difference between the frequencies with the highest
+                power spectral density for each series
+            bounds: [-inf, inf].
+        defpar \rm phase:
+            desc: Common phase offset.
+            init_guess: Zero
+            bounds: [-pi, pi].
+    """
+    # pylint: enable=line-too-long
+
+    def __init__(self):
+        super().__init__(
+            models=[
+                lmfit.models.ExpressionModel(
+                    expr="amp * exp(-x / tau) * cos(2 * pi * (freq - zz / 2) * x + phase + pi) + base",
```

Comment:
Where does the `pi` come from? This is missing in the class documentation.

Reply:
Hmm, it's coming from the fact that cos(0) = 1 and goes down while our sequence does a 2 pi rotation so takes the qubit back to 0 from which it goes up, and we force `amp` to be positive to avoid confusion with `phase`. In 5e29f926, I match the documentation to the model.